docs/testing.md

```diff
@@ -68,9 +68,14 @@ bin/ocis server
 ```
 
 ### Run the acceptance tests
+First we will need to clone the testing app in owncloud which contains the skeleton files required for running the tests.
+In the ownCloud 10 core clone testing app with following command
 
-In the ownCloud 10 core repo run
+```
+git clone https://github.com/owncloud/testing apps/testing
+```
 
+Then run the api acceptance tests with following command.
```

Comment:
Somewhere I also need to define `SKELETON_DIR` which is not mentioned in these instructions:
```
export SKELETON_DIR='/home/phil/git/owncloud/core/apps/testing/data/apiSkeleton'
```
So that needs to be mentioned somewhere.


Also, I put all the "real" core apps into an directory called `apps-external`, that avoids mixing them up with the "built-in" apps in code like `apps/files` and `apps/files_sharing`. So actually I do:

```
git clone https://github.com/owncloud/testing apps-external/testing
export SKELETON_DIR='/home/phil/git/owncloud/core/apps-external/testing/data/apiSkeleton'
```

But for this doc I guess that is a detail that someone can work out if they have put the testing app in `apps-external`